integration/configs.go

```diff
@@ -21,7 +21,17 @@ const (
 	bucketName             = "cortex"
 	cortexConfigFile       = "config.yaml"
 	cortexSchemaConfigFile = "schema.yaml"
-	storeConfigTemplate    = `
+	// Adding an old schema so we test multi schema issues.
```

Comment:
I think this may break the usage in `integration/chunks_storage_backends_test.go`. Given the `chunks_storage_backends_test.go` already builds a multi schema config, I'm not really sure you need to add this. I think we're already covered there. Am I missing anything?